examples/compiled/bar_argmax.vg.json

```diff
@@ -0,0 +1,110 @@
+{
+  "$schema": "https://vega.github.io/schema/vega/v5.json",
+  "description": "The production budget of the movie that has the highest US Gross in each major genre.",
+  "autosize": "pad",
+  "padding": 5,
+  "width": 200,
+  "style": "cell",
+  "data": [
+    {
+      "name": "source_0",
+      "url": "data/movies.json",
+      "format": {"type": "json"},
+      "transform": [
+        {
+          "type": "aggregate",
+          "groupby": ["Major_Genre"],
+          "ops": ["argmax"],
+          "fields": ["US_Gross"],
+          "as": ["argmax_US_Gross"]
+        }
+      ]
+    }
+  ],
+  "signals": [
+    {"name": "y_step", "value": 20},
+    {
+      "name": "height",
+      "update": "bandspace(domain('y').length, 0.1, 0.05) * y_step"
+    }
+  ],
+  "marks": [
+    {
+      "name": "marks",
+      "type": "rect",
+      "style": ["bar"],
+      "from": {"data": "source_0"},
+      "encode": {
+        "update": {
+          "fill": [
+            {
+              "test": "datum[\"argmax_US_Gross\"].Production_Budget === null || isNaN(datum[\"argmax_US_Gross\"].Production_Budget)",
+              "value": null
+            },
+            {"value": "#4c78a8"}
+          ],
+          "tooltip": {
+            "signal": "{\"Production_Budget for max US_Gross\": format(datum[\"argmax_US_Gross\"].Production_Budget, \"\"), \"Major_Genre\": ''+datum[\"Major_Genre\"]}"
+          },
+          "x": {"scale": "x", "field": "argmax_US_Gross.Production_Budget"},
+          "x2": {"scale": "x", "value": 0},
+          "y": {"scale": "y", "field": "Major_Genre"},
+          "height": {"scale": "y", "band": true}
+        }
+      }
+    }
+  ],
+  "scales": [
+    {
+      "name": "x",
+      "type": "linear",
+      "domain": {
+        "data": "source_0",
+        "field": "argmax_US_Gross.Production_Budget"
+      },
+      "range": [0, {"signal": "width"}],
+      "nice": true,
+      "zero": true
+    },
+    {
+      "name": "y",
+      "type": "band",
+      "domain": {"data": "source_0", "field": "Major_Genre", "sort": true},
+      "range": {"step": {"signal": "y_step"}},
+      "paddingInner": 0.1,
+      "paddingOuter": 0.05
+    }
+  ],
+  "axes": [
+    {
+      "scale": "x",
+      "orient": "bottom",
+      "grid": false,
+      "title": "Production_Budget for max US_Gross",
```

Comment:
Production_Budget for max US_Gross -- is the title good enough?

Reply:
It's not quite correct (because we are looking at the record, not the value) but it's the shortest and most reasonable title. So yes, it's good.